sulu/article-bundle/2.0/config/packages/sulu_article.yaml

```diff
@@ -0,0 +1,28 @@
+sulu_article:
+    index_name: '%env(resolve:ELASTICSEARCH_INDEX)%'
+    hosts:
+        - '%env(resolve:ELASTICSEARCH_HOST)%'
+    types:
+        article:
+            translation_key: "sulu_article.article"
+
+sulu_route:
+    mappings:
+        Sulu\Bundle\ArticleBundle\Document\ArticleDocument:
+            generator: schema
+            options:
+                route_schema: '/articles/{object.getTitle()}'
+        Sulu\Bundle\ArticleBundle\Document\ArticlePageDocument:
+            generator: article_page
+            options:
+                route_schema: '{translator.trans("page")}-{object.getPageNumber()}'
+                parent: '{object.getParent().getRoutePath()}'
+
+ongr_elasticsearch:
+    analysis:
+        tokenizer:
+            pathTokenizer:
```

Comment:
The `tokenizer` and `analyzer` key expects a collection of different config. You make up the name `pathTokenizer` and `pathAnalyzer` specifically for this bundle, right? 

If so, this is fine.

https://github.com/ongr-io/ElasticsearchBundle/blob/master/DependencyInjection/Configuration.php#L105

Reply:
Yes, we made up the name of the tokenizer and the analyzer by ourselves, it just needs to be the same as configured in the [ArticleViewDocument](https://github.com/sulu/SuluArticleBundle/blob/master/Document/ArticleViewDocument.php#L68). Though we can't change those right now, because it would be a bc break.

Reply:
Cool cool cool. 

Thank you

Reply:
thank you 🙂